LICENSE.md

```diff
@@ -0,0 +1,27 @@
+# Disclaimer
+
+This license applies only to the source code that is directly maintained in
+this git repository, it does not extend to dependencies from outside of this
+repository, to include other projects owned and/or maintained by Kasm Technologies.
```

Comment:
I think we should be careful about clarifying disclaimers. I wouldn't want to add a disclaimer to a license that isn't legally tested and comes with some unintended side effect. 

I'm not a lawyer (obviously lol) but, to me, this sounds like the license ONLY applies to the source code in THIS repo. Meaning, someone could take the entire source code out of this repo and release it as proprietary software. I want to make sure that derivative works are also guaranteed open-source. Also, I don't think we need to explicitly call out Kasm Technologies here.

Reply:
We can't really impose derivative works be licensed under an open-source license with MIT, if that's a concern; MIT is a permissive license that would allow any derivative works to be proprietary.

Reply:
The verbiage regarding the disclaimer was pulled directly from the upstream Kasm license https://github.com/kasmtech/workspaces-core-images/blob/develop/LICENSE.md
I'd be hesitant to modify it without consulting a lawyer